README.md

```diff
@@ -83,6 +83,7 @@ The devices below are fully feature supported by OpenRazer, which means all avai
 | Razer Blade 15 Advanced (Mid 2021)            |  1532:0276  |
 | Razer Blade 17 Pro (Mid 2021)                 |  1532:0279  |
 | Razer BlackWidow V3 Tenkeyless                |  1532:0A24  |
+| Razer Blade 15 Base (Early 2021)              |  1532:026F  |
```

Comment:
Please keep the list sorted by USB PID, so put this after `Razer Blade 15 Advanced (Early 2021)`

Reply:
Changes made based on feedback.